opensearch_py_ml/ml_commons/ml_commons_client.py

```diff
@@ -56,19 +58,30 @@ def upload_model(
         :type model_config_path: string
         :param isVerbose: if isVerbose is true method will print more messages. default False
         :type isVerbose: boolean
+        :param load_model: Whether to load the model in memory using uploaded model chunks
```

Comment:
Not related to this PR, just remind something we should do.
From 2.7 we are going to rename the APIs https://github.com/opensearch-project/ml-commons/issues/823. We should also change in py-ml

Reply:
Yeah, we have issue open for this: https://github.com/opensearch-project/opensearch-py-ml/issues/122